opencl-aot/source/utils.cpp

```diff
@@ -240,6 +246,7 @@ getOpenCLPlatform(DeviceType Type) {
          DeviceTypesToSupportedPlatformNames[Type]) {
       ErrorMessage += "  " + SupportedPlatformName + '\n';
```

Comment:
It may be also unclear from these error messages that we couldn't find a device of requested type (like CPU). E.g., the OpenCL platform is actually exists, but it doesn't have OpenCL devices with type CPU